layers/+tools/lsp/README.org

```diff
@@ -157,6 +157,9 @@ To display the current project, current file, and document symbols,
                 '((lsp :variables lsp-headerline-breadcrumb-segments '(project file symbols))))
 #+END_SRC
 
+You may need to run =all-the-icons-install-fonts= if you have `all-the-icons` package installed,
```

Comment:
Did you mean to use backticks here? Since this is an Org buffer, perhaps you meant tildes?

Reply:
Oh, my fault..
Updated! Thanks for your review! @Kazark